app/assets/stylesheets/boards.scss

```diff
@@ -0,0 +1,34 @@
+.board {
+ 	width: 95%;
+  height: 95%;
+  border: 3px solid #333;
+  margin: 10px auto 10px auto;
+  vertical-align: middle;
+}
+body {
+ 	padding-top: 10%; 
+ 	padding-bottom: 10%;
+}
+td {
+  empty-cells: show;
+}
+table.one, .board {
+  border-collapse: collapse;
+ 	height: 45em;
+ 	width: 45em;
+ 	overflow:hidden
+}
+.board tr.odd td:nth-child(odd){
+ 	background-color: #000608
+}
+.board tr.even td:nth-child(even){
+ 	background-color: #000608;
+}
+.container{
+ 	width:auto;
+  height:auto;
+  max-width:90%;
+  display: table;
+}
```

Comment:
`.container` is one of the core classes provided by Bootstrap and will ultimately be used all over the site. You shouldn't be modifying its properties directly because other people will use it elsewhere with the expectation that it will conform to the Bootstrap standards.